lib/eventboss/queue_listener.rb

```diff
@@ -1,9 +1,34 @@
 module Eventboss
   class QueueListener
     class << self
+      def list_active
+        listeners = list.values.map(&:name)
+        if Eventboss.configuration.listeners[:include]
```

Comment:
I'd decouple configuration from the rest as much as possible - I know it's not the reality now, but should be our goal.

Why not instead received `include:` and `exclude:` as keyword arguments and then, whoever calls it, pass from configuration? It will be easier to test and work w/ it

Reply:
Added `include:` and `exclude:` arguments defaulting to config.